dashboards/authentication/di-auth-ticf/di-auth-ticf.json.tpl

```diff
@@ -0,0 +1,587 @@
+{
+  "metadata": {
+  },
+  "dashboardMetadata": {
```

Comment:
It looks like lines 4-9 can be removed, maybe a copy/paste error?

Reply:
Eek, sorry good spot! Fixed